android/src/main/java/com/reactnativestripesdk/PaymentMethodCreateParamsFactory.kt

```diff
@@ -145,21 +145,22 @@ class PaymentMethodCreateParamsFactory(
 
   @Throws(PaymentMethodCreateParamsException::class)
   private fun createSepaPaymentSetupParams(): ConfirmSetupIntentParams {
-    val billingDetails = billingDetailsParams?.let { it } ?: run {
-      throw PaymentMethodCreateParamsException("You must provide billing details")
-    }
-    val iban = getValOr(params, "iban", null)?.let { it } ?: run {
-      throw PaymentMethodCreateParamsException("You must provide IBAN")
-    }
+    billingDetailsParams?.let {
+      val iban = getValOr(params, "iban", null) ?: run {
+        throw PaymentMethodCreateParamsException("You must provide IBAN")
+      }
 
-    val sepaParams = PaymentMethodCreateParams.SepaDebit(iban)
-    val createParams =
-      PaymentMethodCreateParams.create(sepaDebit = sepaParams, billingDetails = billingDetails)
+      val sepaParams = PaymentMethodCreateParams.SepaDebit(iban)
+      val createParams =
+        PaymentMethodCreateParams.create(sepaDebit = sepaParams, billingDetails = it)
```

Comment:
nit: below we put each of the parameters on different lines, just for styling it might be nice to do the same here
```suggestion
        PaymentMethodCreateParams.create(
                     sepaDebit = sepaParams, 
                     billingDetails = it
        )
```